pennylane/drawer/drawable_layers.py

```diff
@@ -51,19 +51,15 @@ def _recursive_find_layer(layer_to_check, op_occupied_wires, occupied_wires_per_
     return _recursive_find_layer(layer_to_check - 1, op_occupied_wires, occupied_wires_per_layer)
 
 
-def _recursive_find_mcm_stats_layer(
-    layer_to_check, op_occupied_cwires, used_cwires_per_layer, bit_map
-):
+def _recursive_find_mcm_stats_layer(layer_to_check, op_occupied_cwires, used_cwires_per_layer):
```

Comment:
This function was not using `bit_map` at all, so I removed it.
Fun fact: If a recursive function has excess arguments, pylint won't find out ;-)